src/main/java/net/minecraftforge/client/event/RenderNameplateEvent.java

```diff
@@ -32,33 +35,47 @@
  * {@link #nameplateContent} contains the content being rendered on the name plate/tag. This can be changed by mods.<br>
  * {@link #originalContent} contains the original content being rendered on the name plate/tag. This cannot be
  * changed by mods.<br>
+ * {@link #entityRenderer} contains the entity renderer instance that renders the name plate/tag. This cannot be
+ * changed by mods.<br>
  * {@link #matrixStack} contains the matrix stack instance involved in rendering the name plate/tag. This cannot
  * be changed by mods.<br>
  * {@link #renderTypeBuffer} contains the render type buffer instance involved in rendering the name plate/tag.
  * This cannot be changed by mods.<br>
+ * {@link #packedLight} contains the sky and block light values used in rendering the name plate/tag.<br>
  * <br>
  * This event has a result. {@link HasResult}. <br>
  * ALLOW will force-render name plate/tag, DEFAULT will ignore the hook and continue using the vanilla check
  * & DENY will prevent name plate/tag from rendering<br>
  * <br>
  * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
  **/
+@SuppressWarnings("rawtypes")
```

Comment:
Use `EntityRenderer<?>` instead of raw `EntityRenderer`, don't just suppress the warning